packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignSelf: 'baseline',
+    borderStyle: 'solid',
+    borderWidth: 2,
+    flex: 0,
+    padding: 4,
+    borderRadius: 999,
+    // marginRight: 'auto',
+    // marginLeft: 'auto',
+    top: 8,
+    position: 'relative',
+    boxShadow: theme.shadows[2],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="default"`. */
+  defaultDefault: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: theme.palette.grey[300],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
+  outlinedDefalut: {
```

Comment:
```suggestion
  outlinedDefault: {
```

(Although I think the plan is to move away from using `default` as the default value...)

Reply:
replaced default color with grey